Makefile

```diff
@@ -2,7 +2,7 @@ CWD = $(shell pwd)
 SRC_DIR = ${CWD}/pootle
 DOCS_DIR = ${CWD}/docs
 STATIC_DIR = ${SRC_DIR}/static
-ASSETS_DIR = $(shell python -c "from pootle.settings import *; print(STATIC_ROOT)")
+ASSETS_DIR = ${SRC_DIR}/assets
```

Comment:
Please add a comment to mention this refers to the value of `STATIC_ROOT` and that it's hard-coded here.